lineage/figures/figure9.py

```diff
@@ -19,34 +19,24 @@ def makeFigure():
     """
     ax, f = getSetup((7, 3), (1, 2))
 
-    data = [Lapatinib_Control + Gemcitabine_Control, Lapt25uM, Lapt50uM, Lap250uM, Gem5uM, Gem10uM, Gem30uM]
-    dataFull = []
-
-    # Find the cell cycle shape parameters to be set as constant from the one state model
-    tHMM_solver = tHMM(X=data[0], num_states=1)
-    tHMM_solver.fit()
-
-    constant_shape = [int(tHMM_solver.estimate.E[0].params[2]), int(tHMM_solver.estimate.E[0].params[4])]
-
-    # Set shape
-    for population in data:
-        for lin in population:
-            for E in lin.E:
-                E.G1.const_shape = constant_shape[0]
-                E.G2.const_shape = constant_shape[1]
-
-    # Copy out data to full set
-    for _ in desired_num_states:
-        dataFull = dataFull + deepcopy(data)
-
-    # Run fitting
-    output = run_Analyze_over(dataFull, np.repeat(desired_num_states, len(data)))
-    AICs = np.array([oo[0].get_AIC(oo[2])[0] for oo in output])
-    AICs = np.reshape(AICs, (desired_num_states.size, len(data)))
-    AICs -= np.min(AICs, axis=0)
-
-    lapAIC = np.sum(AICs[:, 0:4], axis=1)
-    gemAIC = np.sum(AICs[:, np.array([0, 4, 5, 6])], axis=1)
+    lapatinib = [Lapatinib_Control + Gemcitabine_Control, Lapt25uM, Lapt50uM, Lap250uM]
+    gemcitabine = [Lapatinib_Control + Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM]
+
+    def find_AIC(data, desired_num_states):
+        # Copy out data to full set
+        dataFull = []
+        for _ in desired_num_states:
+            dataFull.append(data)
+
+        # Run fitting
+        output = run_Analyze_over(dataFull, desired_num_states)
+        AICs = np.array([oo[0][0].get_AIC(oo[2])[0] for oo in output])
```

Comment:
Doesn't this need `atonce=True`?

Reply:
I removed that argument and instead of it I used `isinstance(population, list)`. I thought you meant this

Reply:
Yes, but you have an if statement in `get_AIC`.

Reply:
Oh that.. sorry, I'll fix it.